frontend/src/components/Login.js

```diff
@@ -37,6 +38,9 @@ class Login extends Component {
        }
     }).catch(() => {
         console.log('unauthorized login');
```

Comment:
You'll encounter a lot of temporary console logs, TODOs and such which I left around. Feel free to remove them when there is no need for them any more, like here